Content.Client/Atmos/UI/GasThermomachineBoundUserInterface.cs

```diff
@@ -50,7 +54,12 @@ private void OnToggleStatusButtonPressed()
 
         private void OnTemperatureChanged(float value)
         {
-            var actual = Math.Clamp(value, _minTemp, _maxTemp);
+            var actual = 0f;
```

Comment:
Why does the client need to check min/max temperature? Isn't this checked and clamped on the server?

Reply:
so that it immediately clamps as opposed to having to wait for it to sync, apparently (tested this ingame)